package.json

```diff
@@ -31,20 +31,20 @@
     "react-dom": ">=16.8.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.2.3",
-    "@babel/core": "^7.3.4",
-    "@babel/preset-env": "^7.3.4",
+    "@babel/cli": "^7.6.0",
+    "@babel/core": "^7.6.0",
+    "@babel/preset-env": "^7.6.0",
     "@babel/preset-react": "^7.0.0",
-    "browserslist-config-canopy": "^2.0.0",
-    "react": "^16.8.4",
-    "react-dom": "^16.8.4",
-    "rimraf": "^2.6.3",
-    "rollup": "^1.5.0",
-    "rollup-plugin-babel": "^4.3.2",
-    "rollup-plugin-babel-minify": "^7.0.0",
-    "rollup-plugin-node-resolve": "^4.0.1"
+    "browserslist-config-canopy": "^3.1.2",
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
+    "rollup": "^1.21.4",
+    "rollup-plugin-babel": "^4.3.3",
+    "rollup-plugin-babel-minify": "^9.0.0",
+    "rollup-plugin-node-resolve": "^5.2.0"
   },
   "dependencies": {
-    "@types/react": "^16.8.23"
+    "@types/react": "^16.9.2",
+    "rimraf": "^3.0.0"
```

Comment:
rimraf should be a devDependency, not a dependency. This is so people who install bandicoot don't also install rimraf.

Reply:
🤦‍♂ I accidentally removed it thinking it was not used and then put it back wrong.